jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/OdeSvcsApplication.java

```diff
@@ -50,7 +56,13 @@ public static void main(String[] args) throws MalformedObjectNameException, Inte
       mbs.registerMBean(mBean, name);
 
       SecurityProtocols.getInstance().addAuthenticationProtocol(new AuthSHA());
+      SecurityProtocols.getInstance().addAuthenticationProtocol(new AuthHMAC128SHA224());
+      SecurityProtocols.getInstance().addAuthenticationProtocol(new AuthHMAC192SHA256());
+      SecurityProtocols.getInstance().addAuthenticationProtocol(new AuthHMAC256SHA384());
+      SecurityProtocols.getInstance().addAuthenticationProtocol(new AuthHMAC384SHA512());
       SecurityProtocols.getInstance().addAuthenticationProtocol(new AuthMD5());
+      SecurityProtocols.getInstance().addPrivacyProtocol(new PrivAES128());

```

Comment:
Do these protocols automatically get used or do we have to do anything further to utilize them?

Reply:
In SnmpSession.java lines 108-109 those protocols are utilized when creating the SNMP user.